buffer/src/test/java/io/netty/buffer/AbstractPooledByteBufTest.java

```diff
@@ -34,4 +38,26 @@ protected ByteBuf newBuffer(int length, int maxCapacity) {
         assertEquals(0, buffer.readerIndex());
         return buffer;
     }
+
+    @Test
+    public void ensureWritableWithEnoughSpaceShouldNotThrow() {
+        ByteBuf buf = buffer(1);
```

Comment:
Also you should use enough bytes to be sure that the buffer is big enough and not depend on the impl detail that we will use big enough buffer when we request a size of 1. Use `4` or something like that.